arrow/Cargo.toml

```
@@ -86,6 +89,8 @@ rand = { version = "0.8", default-features = false, features = ["std", "std_rng"
 criterion = { version = "0.3", default-features = false }
 flate2 = { version = "1", default-features = false, features = ["rust_backend"] }
 tempfile = { version = "3", default-features = false }
+lz4 = { version = "1.23", default-features = false }
+zstd = { version = "0.11", default-features = false }
```

The version is different from above?

```suggestion
zstd = { version = "0.11.1", default-features = false }
```

---

🤔  now that I see that it seems like there is no reason to have these dependencies left in both dependencies and dev-dependencies. I will remove them from dev-dependences.

Thanks for the catch